RevenueCatUI/UIKit/PaywallFooterViewController.swift

```diff
@@ -51,6 +51,25 @@ public final class PaywallFooterViewController: PaywallViewController {
     public required init?(coder: NSCoder) {
         fatalError("init(coder:) has not been implemented")
     }
+
+    public override func loadView() {
+        super.loadView()
+
+        self.addChild(self.hostingController)
+        self.view.addSubview(self.hostingController.view)
```

Comment:
The parent class is already doing this though?

Reply:
🤦 I was rushing through things and missed it, thanks